src/vivarium/framework/results/context.py

```diff
@@ -115,20 +111,20 @@ def add_stratification(
         Parameters
         ----------
         name
-            Name of the column created by the `mapper`.
+            Name of the stratification.
         sources
             A list of the columns and values needed as input for the `mapper`.
         categories
-            List of string values that the `mapper` is allowed to map to.
+            Exhaustive list of all possible stratification values.
         excluded_categories
             List of possible stratification values to exclude from results processing.
             If None (the default), will use exclusions as defined in the configuration.
         mapper
-            A callable that takes inputs from the columns and pipelines specified by
-            `requires_columns` and `requires_values` arguments and produces a
-            Series. All values produced by a `mapper` must be one of the elements of
-            `categories`. A simulation will fail if the `mapper` ever produces
-            an invalid value.
+            A callable that maps the columns and value pipelines specified by the
+            `requires_columns` and `requires_values` arguments to the stratification
```

Comment:
`requires_columns` and `requires_values` are not arguments to this method. You should refer to `sources` instead.